SPEC.md

```diff
@@ -105,12 +105,23 @@ The test index makes use of symbolic language in describing connection and messa
 
 ### ZG-CONFORMANCE-004
     
-    The node responds with mtLEDGER_DATA for mtGET_LEDGER.
+    The node responds with mtLEDGER_DATA for mtGET_LEDGER with iType == LiBase.
     
     <>
     -> mtGET_LEDGER with iType == LiBase
     <- mtLEDGER_DATA
-    or:
 
+### ZG-CONFORMANCE-005
+
+    The node responds with mtLEDGER_DATA for mtGET_LEDGER with iType == LiAsNode.
+
+    <>
     -> mtGET_LEDGER with iType == LiAsNode
     <- mtLEDGER_DATA
+
+### ZG-CONFORMANCE-006
+
+    The node requests mtGET_PEER_SHARD_INFO_V2 after start up.
+
+    ->
```

Comment:
Does this include the handshake or is that message sent immediately?

Reply:
Looking at the code it seems like it's after the handshake.

Reply:
We might want to find a way to make that clear in the symbolic language if we ever have cases that don't include the handshake, for now it's fine 👍